tools/vagrant-box-atom/Gemfile.lock

```diff
@@ -0,0 +1,18 @@
+GEM
+  remote: https://rubygems.org/
+  specs:
+    excon (0.80.1)
+    log4r (1.1.10)
+    vagrant_cloud (3.0.4)
+      excon (~> 0.73)
+      log4r (~> 1.1.10)
+
+PLATFORMS
+  ruby
+  x86_64-linux
```

Comment:
I guess it doesn't really matter since you're not publishing anything but shouldn't this platform be a "darwin" something?

Reply:
TBH, I have no idea, this is what `bundle install` created and seems to be working in both Linux and MacOS.